edit/lint.js

```diff
@@ -0,0 +1,319 @@
+/* global CodeMirror CSSLint editors makeSectionVisible showHelp showCodeMirrorPopup */
+/* global stylelintDefaultConfig onDOMscripted injectCSS require */
+'use strict';
+
+function initLint() {
+  document.getElementById('lint-help').addEventListener('click', showLintHelp);
+  document.getElementById('lint').addEventListener('click', gotoLintIssue);
+  window.addEventListener('resize', resizeLintReport);
+  document.getElementById('stylelint-settings').addEventListener('click', openStylelintSettings);
+
+  // touch devices don't have onHover events so the element we'll be toggled via clicking (touching)
+  if ('ontouchstart' in document.body) {
+    document.querySelector('#lint h2').addEventListener('click', toggleLintReport);
```

Comment:
replace all document.querySelector with `$` and element.querySelector with `$(selector, element)` - at least in the new code.